package.json

```diff
@@ -28,6 +28,8 @@
     "apollo-server-restify": "^1.3.2",
     "bluebird": "^3.5.1",
     "dataloader": "^1.3.0",
+    "dotenv": "^5.0.0",
+    "file-system": "^2.2.2",
```

Comment:
why do we need this lib?

Reply:
I use this to check if log directory exist and if it doesn't exist, create it